packages/components/src/components/telekom/telekom-header/telekom-header.tsx

```diff
@@ -33,6 +33,10 @@ export class TelekomHeader {
   @Prop() appNameLink?: string;
   @Prop() appNameClick?: any;
   @Prop() type: string = 'default';
+  @Prop() metaNavAriaLabel?: string = 'Meta navigation';
+  @Prop() metaNavExternalAriaLabel?: string = 'External meta navigation';
+  @Prop() languageSwitcherAriaLabel?: string = 'Language switcher';
```

Comment:
We could rename this to `langSwitcherAriaLabel` for consistency with the `lang-switcher` part. What do you think? (if yes, there's many other places to be updated…)